explorer/.env.sample

```diff
@@ -19,6 +19,12 @@ DISCORD_BOT_TOKEN="<discord-bot-token>"
 
 # Subspace Discord Server Role ID's
 DISCORD_GUILD_ROLE_ID_FARMER="<discord-farmer-role-id>"
+DISCORD_GUILD_ROLE_ID_OPERATOR="<discord-operator-role-id>"
+DISCORD_GUILD_ROLE_ID_NOMINATOR="<discord-nominator-role-id>"
+
+# GitHub integration for authentication
+GITHUB_CLIENT_ID="<github-client-id>"
```

Comment:
Is this a specific Repo ID we provide here, or we can pick the whole organization? We likely want to give labels for the multiple repo contributors.